pkg/webhooks/mutator/hyperConvergedMutator_test.go

```diff
@@ -264,9 +265,17 @@ var _ = Describe("test HyperConverged mutator", func() {
 
 			AfterEach(func() {
 				util.GetClusterInfo = getClusterInfo
+
```

Comment:
As a follow up of #3205, please remove line 264 and the whole `AfterEach` method, as this is not needed anymore (after removing the new code from `AfterEach`).